src/main/java/seedu/address/model/person/Person.java

```diff
@@ -19,7 +19,8 @@ public class Person {
     // Identity fields
     private final StudentId studentId;
     private final Name name;
-    private final Phone phone;
+    private final Phone parentPhoneOne;
```

Comment:
It is good to have two separate Phone numbers👍 However, what if a student only have one parent?